docs/src/main/asciidoc/_high-availability/subsystem-support/Infinispan_Subsystem.adoc

```diff
@@ -445,6 +453,19 @@ e.g. To configure a non-blocking state transfer:
 
 For a complete list of state-transfer attributes, refer to the link:wildscribe[WildFly management model documentation^]
 
+
+==== Injecting a cache into Jakarta EE applications
+
+A cache can be injected directly into Jakarta EE applications using the `@Resource` annotation, e.g.
+
+[source,java,options="nowrap"]
+----
+@Resource(lookup = "java:jboss/infinispan/cache/foo/bar") // <1>
+private org.infinispan.Cache<Integer, Object> cache;
+----
+<1> where `foo` is the name of the cache container and `bar` is the name of the cache to inject
+
+
```

Comment:
Additionally, `@Resource(lookup = "java:jboss/infinispan/cache/foo/default")` will inject the *default* cache of the "foo" container.